code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -9,15 +9,15 @@ LINEN BINS
 	desc = "A surprisingly soft linen bedsheet."
 	icon = 'icons/obj/items.dmi'
 	icon_state = "sheet"
-	item_state = "bedsheet"
+	item_state = "sheet"
 	layer = 4.0
```

Comment:
You can also remove this one entirely, same as above. There's nothing on `/obj/item` so it'll default to using `item_state` for the inhand, which in this case is exactly the same.
```suggestion
	layer = 4.0
```